test/EnsembleKalmanProcess/runtests.jl

```diff
@@ -26,13 +26,16 @@ const EKP = EnsembleKalmanProcesses
 
     ### Generate data from a linear model: a regression problem with n_par parameters
     ### and 1 observation of G(u) = A \times u, where A : R^n_par -> R^n_obs
-    n_obs = 10                  # dimension of synthetic observation from G(u)
+    n_obs = 30                  # dimension of synthetic observation from G(u)
     n_par = 2                  # Number of parameteres
     u_star = [-1.0, 2.0]          # True parameters
-    noise_level = 0.05            # Defining the observation noise level (std) 
+    noise_level = 0.1             # Defining the observation noise level (std) 
     # Test different AbstractMatrices
     Γy_vec =
         [noise_level^2 * I, noise_level^2 * Matrix(I, n_obs, n_obs), noise_level^2 * Diagonal(Matrix(I, n_obs, n_obs))]
+    # Test different localizers
+    loc_methods = [RBF(2.0), Delta(), NoLocalization()]
+
```

Comment:
What do you mean by damping here? Do you want calibration tests, or just check some corollary of Schur product theorem based on the localized/non-localized covariances?

Reply:
Nothing so fancy, just a test of the localization kernels directly outside of the EKP. 

i.e take some matrix `M` and do `M .* localization.kernel` for each different kernel type and check the solution == what it should be. This simply checks that the methods are built properly and demonstrates they work in the exact use case as inside EKP (where we apply `ug_cov .* localization.kernel` ) Otherwise we only have the very indirect checks on the timestep condition `Delta < 1` as a test. (which is fine, but it's not a unit test of the object)

Reply:
This exact unit test is kind of difficult since we don't store the modified `cov_ug` in the EKP, but I have added a unit test checking that any localization matrix on a positive semidefinite matrix boosts the rank of the result.